queries/session.py

```diff
@@ -7,7 +7,10 @@
 from ..authentication_config import MACHINE_ACCOUNTS
 
 SESSION_GRAPH = "http://mu.semte.ch/graphs/sessions"
-ACCOUNT_GRAPH = "http://mu.semte.ch/graphs/system/users" # http://mu.semte.ch/graphs/public for mock-login
+ACCOUNT_GRAPH = (
+    os.environ.get("ACCOUNT_GRAPH") or 
+    "http://mu.semte.ch/graphs/system/users"
+) # http://mu.semte.ch/graphs/public for mock-login
```

Comment:
`or` ?
You can pass the default:
`os.environ.get("ACCOUNT_GRAPH", "http://mu.semte.ch/graphs/system/users")`